src/_utils/item/index.tsx

```diff
@@ -1,13 +1,18 @@
 import styled from 'styled-components'
 
+import { normalizeHorizontally } from '../../layout/layoutNormalizer'
 import { color, space } from '../branding'
 import { Item } from './Item'
 
 const StyledItem = styled(Item)`
+  /* HorizontalNormalization */
+  & {
+    ${normalizeHorizontally}
```

Comment:
by the css cascade if you apply a padding here it will be replace for declations just after... 
so you can move after component declarion or
remove the `&` on css definition for the component and let the & here to bring specificity